sandpack-react/src/components/Tests/Header.tsx

```diff
@@ -34,6 +35,8 @@ interface Props {
   watchMode: boolean;
   setWatchMode: () => void;
   showSuitesOnly: boolean;
+  hideVerboseButton?: boolean;
```

Comment:
Throughout the sandpack components we use the `show` instead of `hide` when we want to switch the visibility of something, could you do that?